matrix/count_paths.py

```diff
@@ -0,0 +1,43 @@
+def dfs(grid: list(list(int)), row: int, col: int, visit: set()) -> int:
```

Comment:
```suggestion
def dfs(grid: list[list[int]], row: int, col: int, visit: set) -> int:
```
Is this what you mean?